extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -514,6 +520,16 @@ export class InstrumentationService {
     this.win_.setTimeout(this.win_.clearInterval.bind(this.win_, intervalId),
         maxTimerLength * 1000);
   }
+
+  isTriggerAllowed_(triggerType, element) {
+    if (!isInIframe(element) ||
```

Comment:
`isInIframe` will return true in search viewer mode, which probably is not what you want.

I think there's plan to have different attribute  in the root HTML tag, something like:

```
<html amp-a4a>
```

@Gregable  is this still the plan?